gulp/performance-tests.js

```diff
@@ -58,3 +55,26 @@ gulp.task('performance-tests', ['all-tests-with-no-performance'], function () {
 		}, true)));
         */
 });
+
+var perfTestsPath = './performance-tests/tests';
+var files = fs.readdirSync(perfTestsPath);
+files.forEach(function (file) {
+	if (!fs.lstatSync(path.join(perfTestsPath, file)).isDirectory()) return;
+
+	gulp.task('performance-scenario-' + file, [], function () {
```

Comment:
what will be the name of the task? performance-scenario-computedDiamond1000UpdatesWith500Dependencies ?